corehq/apps/accounting/tests/test_credit_lines.py

```diff
@@ -308,7 +309,7 @@ def test_transfers(self):
             edition=SoftwarePlanEdition.STANDARD
         )
         first_sub = Subscription.new_domain_subscription(
-            self.account, self.domain, advanced_plan
+            self.account, self.domain.name, advanced_plan
```

Comment:
Did your change affect this test? If not, does Travis run this test? And if not, should it?